catalog/app/data/workflows.spec.js

```diff
@@ -0,0 +1,108 @@
+import parseWorkflows, {
+  emptyWorkflowsConfig,
+  workflowNotAvaliable,
+  workflowNotSelected,
+} from './workflows'
+
+describe('Workflows config', () => {
+  it('empty file', () => {
+    const config = parseWorkflows('')
+    expect(config).toBe(emptyWorkflowsConfig)
+    expect(config.workflows[0].slug).toBe(workflowNotAvaliable)
```

Comment:
i think this assertion is redundant, because the previous one already guarantees this

Reply:
It guarantees it since `workflowNotAvaliable` is a part of `emptyWorkflowsConfig`. You can break it, and this test to stop you from doing this